src/Fixer/ControlStructure/NoBreakCommentFixer.php

```diff
@@ -78,7 +78,7 @@ public function getDefinition(): FixerDefinitionInterface
      */
     public function isCandidate(Tokens $tokens): bool
     {
-        return $tokens->isAnyTokenKindsFound([T_CASE, T_DEFAULT]);
+        return $tokens->isTokenKindFound(T_SWITCH);
```

Comment:
not a bug, but on PHP8.1 and up `case` and `default` can be used outside `switch` statements, as such this check is better